src/lib/flows/import-from-csv/parse-upload.ts

```diff
@@ -0,0 +1,75 @@
+import { parseFile as parseCsv } from '$lib/utils/csv';
+
+/**
+ * Determine if at least on of the specified header names are in the header
+ * of the given parsed CSV file.
+ *
+ * @param parsedFile A parsed CSV file.
+ * @param csvHeaders A list of header names.
+ * @returns True if at least on the specified header names exists in the
+ *  header, false otherwise.
+ */
+function hasHeader(parsedFile: Array<Array<string>>, csvHeaders: Array<string>): boolean {
```

Comment:
not a hard rule but we usually use e.g. `string[]` syntax to denote array types, rather than `Array<string>`. Bit more compact 👍

Reply:
Ok, I'll update this